Agenda Maker/src/code.js

```diff
@@ -0,0 +1,146 @@
+
+/*
+See if the folder of Agenda docs exists, or otherwise create it.
+*/
+function checkFolder (){
+  var folders = DriveApp.getFoldersByName("Agenda Maker - App");
+  //Find the folder if it exists
+  while(folders.hasNext()){
+    var folder = folders.next();
+    if ((folder.getDescription() == "App Script App - Do not change this description") && (folder.getOwner().getEmail()==Session.getActiveUser().getEmail()))
```

Comment:
*Apps

Reply:
Can you clarify the requested change for this line of code?